leo/core/leoCommands.py

```diff
@@ -53,8 +53,8 @@
     from leo.commands.keyCommands import KeyHandlerCommandsClass
     from leo.commands.killBufferCommands import KillBufferCommandsClass
     from leo.commands.rectangleCommands import RectangleCommandsClass
-    from leo.commands.rstCommands import RstCommands
     from leo.commands.spellCommands import SpellCommandsClass
+    from leo.core.rstCommands import RstCommands
```

Comment:
this line should be: 
    from leo.core.leoRst import RstCommands

Reply:
@boltex Fixed at d9eec37.